src/content/docs/style-guide/writing-docs/writer-workflow/set-up-local.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: Set up your local build environment
+topics:
+  - Tech writer style guide
+  - Processes and procedures
+---
+
+Running the site locally makes testing and previewing large changes much easier. There are several things you need to do get this working. Fortunately, you only need to do this once.
+
+## 0. Enable VS Code in your terminal
+
+It's handy to be able to open files from your terminal directly in VS Code. You may only use this a couple of times, but it's much nicer than trying to edit files in the terminal itself.
+
+1. Open Visual Studio Code.
+2. Press **command + shift + p** to open the command palette, then type "install code", and then click **Shell command: Install 'code' command in PATH**.
+3. In your terminal, type `code <var>filename</var>` to open that file in VS Code.
+
+## 1. Set up GitHub Desktop
```

Comment:
I'd mention that this section also downloads the repo locally. Maybe it's just me, but this header makes it seem like I'm just getting the app set up.